lxd/instances_post.go

```diff
@@ -544,8 +545,8 @@ func createFromCopy(s *state.State, r *http.Request, projectName string, profile
 		return nil
 	}
 
-	resources := map[string][]string{}
-	resources["instances"] = []string{req.Name, req.Source.Source}
+	resources := map[string][]api.URL{}
+	resources["instances"] = []api.URL{*api.NewURL().Path(version.APIVersion, "instances", req.Name, req.Source.Source)}
```

Comment:
Why is `Source` part of the URL path? Previously, these were 2 separate elements. I believe there should be 2 URL paths now.